sunpy/tests/figure_hashes_mpl_dev_ft_261_astropy_dev.json

```diff
@@ -29,7 +29,7 @@
     "sunpy.timeseries.tests.test_timeseriesbase.test_fermi_gbm_peek": "a0addeae9ec0eda509c976e9058b2d90fa748b2c3d4a64df43b3a064d85f34d9",
     "sunpy.timeseries.tests.test_timeseriesbase.test_generic_ts_peek": "73a71e717b7dd0dd345f0f8a8bfa7f06387607417b3ecd123adfd8f630f93d51",
     "sunpy.timeseries.tests.test_timeseriesbase.test_goes_peek": "5e8a20ed592398ac710798e7995ba7da158743d974820508656b2ba3bf62e152",
-    "sunpy.timeseries.tests.test_timeseriesbase.test_lyra_peek": "f97e0dbbbda228673bb7543f962388cdfd8396fcf00cbeb4e5a65559db173da4",
+    "sunpy.timeseries.tests.test_timeseriesbase.test_lyra_peek": "d134646089aa9cb217b798e61fbfbfdcf6acf50a18f9292e95294d2cae6d24ea",
```

Comment:
This change is just because the LYRA plot is now correctly using tight layout. The new figure is
![sunpy timeseries tests test_timeseriesbase test_lyra_peek](https://user-images.githubusercontent.com/6197628/94560821-0f7c0e80-025b-11eb-99af-35511b33919d.png)